languages/kotlin/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,47 @@
+Owner of your local pizza place asked you to help her to count cooking time for different pizzas. In this exercise you're going to write few functions to calculate that.
+
+You have three tasks.
+
+## 1. Calculate remaining baking time in minutes
+
+Define and implement `remainingMinutesInOven()` function that will count how many more minutes you need to keep pizza in the oven.
+
+Function should accept one **optional** parameter (with default value) for the amount of minutes that passed since you've started baking pizza. If parameter is missing - assume that you've just put pizza into the oven and **0 (zero)** minutes passed already.
```

Comment:
```suggestion
The function should accept one **optional** parameter (with default value) for the amount of minutes that passed since you've started baking pizza. If the parameter is missing - assume that you've just put pizza into the oven and **0 (zero)** minutes passed already.
```